ios/FluentUI/People Picker/ContactCollectionViewCell.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class ContactCollectionViewCell: UICollectionViewCell {
+    static var identifier: String { return "ContactCollectionViewCell" }
+    var contactView: ContactView
+
+    override init(frame: CGRect) {
+        contactView = ContactView(title: "", subtitle: "")
+        super.init(frame: frame)
+    }
+
+    func setup(contact persona: PersonaData) {
+        if persona.name != "" {
+            contactView = ContactView(identifier: persona.name)
+        } else {
+            contactView = ContactView(identifier: persona.email)
+        }
+
+        contactView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
I don't think you need this line

Reply:
I think this line is actually needed, but I misplaced it. It was missing from `ContactView.swift` and I'll move it there.